.github/workflows/wheels.yml

```diff
@@ -30,38 +28,72 @@ jobs:
         with:
           path: ./dist
 
-
-  make_wheels:
-    name: ${{ matrix.name }}
+  make_cibw_v1_wheels:
+    name: "cibuildwheel v1: ${{ matrix.name }}"
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        # Only build for platforms that cibw v2 doesn't support
         include:
-        - os: macos-latest
-          build: "cp*-macosx_x86_64"
-          name: macOS Intel
-        - os: windows-latest
-          build: "cp3*-win_amd64"
-          skip: "cp35-win_amd64"
-          name: Windows 64-bit
-        - os: ubuntu-latest
-          build: "cp*-manylinux_x86_64"
-          name: Linux Intel 64-bit
-        - os: ubuntu-latest
-          build: "cp36-manylinux_aarch64"
-          name: Linux Aarch64 3.6
-        - os: ubuntu-latest
-          build: "cp37-manylinux_aarch64"
-          name: Linux Aarch64 3.7
-        - os: ubuntu-latest
-          build: "cp38-manylinux_aarch64"
-          name: Linux Aarch64 3.8
-        - os: ubuntu-latest
-          build: "cp39-manylinux_aarch64"
-          name: Linux Aarch64 3.9
+          - os: macos-latest
+            build: "cp27-macosx_x86_64"
+            name: macOS Intel
+          - os: ubuntu-latest
+            build: "cp27-manylinux_x86_64"
+            name: Linux Intel 64-bit
 
     steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      ## Build
 
+      - uses: pypa/cibuildwheel@v1.12.0
+        env:
+          CIBW_TEST_REQUIRES: pytest numpy
+          CIBW_TEST_COMMAND: pytest {project}/tests
+          CIBW_ARCHS_LINUX: auto aarch64
+          CIBW_BUILD: ${{ matrix.build }}
+
+      - name: Check with Twine
+        run: |
+          pipx run twine check wheelhouse/*
+
+      - name: Upload artifacts to GitHub
+        uses: actions/upload-artifact@v2
+        with:
+          path: wheelhouse/*.whl
+
+  make_cibw_v2_wheels:
+    name: "cibuildwheel v2: ${{ matrix.name }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
+            build: "cp*-macosx_x86_64"
+            name: macOS Intel
+          - os: windows-latest
+            build: "cp3*-win_amd64"
+            name: Windows 64-bit
+          - os: ubuntu-latest
+            build: "cp*-manylinux_x86_64"
+            name: Linux Intel 64-bit
+          - os: ubuntu-latest
+            build: "cp36-manylinux_aarch64"
+            name: Linux Aarch64 3.6
+          - os: ubuntu-latest
+            build: "cp37-manylinux_aarch64"
+            name: Linux Aarch64 3.7
+          - os: ubuntu-latest
+            build: "cp38-manylinux_aarch64"
+            name: Linux Aarch64 3.8
+          - os: ubuntu-latest
+            build: "cp39-manylinux_aarch64"
+            name: Linux Aarch64 3.9
```

Comment:
Does this simplify if we give up `3.5` support?

Reply:
That is, could we now we get away with just something like:

```
  build_wheels:
    name: Build wheels on ${{ matrix.os }}
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ubuntu-20.04, windows-2019, macos-10.15]
```

as in https://cibuildwheel.readthedocs.io/en/stable/setup/#github-actions ?

Reply:
I think it could potentially simplify. There's also a tradeoff here between simplicity and parallelizing the runners. If you break out more matrix options here, you're able to better parallelize the builds.

My idea for this PR was to update to v2 without making substantive changes, which can then be addressed in future PRs

Reply:
> Does this simplify if we give up `3.5` support?

I'm not sure I follow this (or maybe you meant `2.7`?). cibuildwheel v2 doesn't support Python 3.5: https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip

So any 3.5 wheels would still need to be built with cibuildwheel v1.